src/coreclr/jit/emitxarch.cpp

```diff
@@ -13705,6 +13815,43 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
         emitDispIns(id, false, dspOffs, true, emitCurCodeOffs(*dp), *dp, (dst - *dp));
     }
 
+#ifdef FEATURE_LOOP_ALIGN
```

Comment:
Why is this code within `#ifdef DEBUG` block? That's going to lead to Debug/Release code differences

Reply:
It shouldn't be, my bad. I guess the `#endif` got removed after rebase. I will fix it.